packages/@lwc/perf-benchmarks/src/__benchmarks__/ssr/table-render-10k.benchmark.js

```diff
@@ -19,7 +19,7 @@ benchmark(`ssr/table-v2/render/10k`, () => {
 
         return renderComponent(
             'benchmark-table',
-            generateMarkup,
+            Table,
```

Comment:
This is the main reason I wanted to do this. I was trying to port over the "styled component" benchmarks and it was a pain because of dealing with the non-default export only for the SSR case.